templates/admin/awardnominations/view.html.twig

```diff
@@ -0,0 +1,52 @@
+{% extends '@Forumify/admin/base.html.twig' %}
+{% block title_page %}{% endblock %}
+{% block body %}
+    {% if data == null %}
+        <p>Unknown nomination!</p>
+    {% else %}
```

Comment:
Controller should take care of this and throw HttpNotFoundException. This would already be taken care of when using the `AbstractCrudController`